object_store/src/client/backoff.rs

```diff
@@ -18,7 +18,12 @@
 use rand::prelude::*;
 use std::time::Duration;
 
-/// Exponential backoff with jitter
+/// Backoff with decorrelated jitter algorithm
```

Comment:
```suggestion
/// Exponential backoff with decorrelated jitter algorithm
```

This is still an exponential backoff, in that the delay is multiplied each time. This is very important to document IMO, as the non linear backoff is important